app/models/user.rb

```diff
@@ -66,4 +68,19 @@ def delete_identities
       id.destroy
     end
   end
+
+  private
+    # Private: Create an identity for local file handling when user is created.
+    # Future: This should only happen if user has a password.
+    #
+    # Returns nothing
+    def append_local_identity
+      if own_password && !identities.exists?(service: 'local')
+        local_identity = Identity.new(service: 'local',
+                                      token: SecureRandom.hex(5),
+                                      name: 'Bruse',
+                                      uid: SecureRandom.hex(5))
```

Comment:
What happens if _uid_ is equal to some other _uid_ and an error is raised? Perhaps a check to see that the `local_identity` really is created.